creature_factory.cpp

```diff
@@ -1281,6 +1295,18 @@ CreatureAttributes CreatureFactory::getAttributesFromId(CreatureId id) {
           c.chatReactionFriendly = "curses all dungeons"_s;
           c.chatReactionHostile = "\"Die!\""_s;
           c.name = "knight";);
+    case CreatureId::DARK_KNIGHT: 
+      return CATTR(
+          c.viewId = ViewId::DARK_KNIGHT;
+          c.attr = LIST(40_dam, 40_def, 110_spd );
+          c.body = Body::humanoid(Body::Size::LARGE);
+          //You really need to defeat this guy with magic.
+          //He is an expert at conventional fighting.
+          c.permanentEffects[LastingEffect::MELEE_RESISTANCE] = 1;
+          c.permanentEffects[LastingEffect::RANGED_RESISTANCE] = 1;
+          c.chatReactionFriendly = "curses all castles"_s;
+          c.chatReactionHostile = "\"Face death!\""_s;
+          c.name = "The Dark Knight";);
```

Comment:
Please remove "The", otherwise there will be a double "the" everywhere in the messages.